src/libs/antares/study/area/list.cpp

```diff
@@ -335,7 +335,23 @@ bool saveAreaAdequacyPatchIniFile(const Area& area, const Clob& buffer)
 {
     IniFile ini;
     IniFile::Section* section = ini.addSection("adequacy-patch");
-    section->add("use-adequacy-patch", area.bUseAdequacyPatch);
+    int value;
+    switch (area.adequacyPatchMode)
+    {
+    case Data::adqmVirtualArea:
+        value = 0;
+        break;
+    case Data::adqmPhysicalAreaOutsideAdqPatch:
+        value = 1;
+        break;
+    case Data::adqmPhysicalAreaInsideAdqPatch:
+        value = 2;
+        break;
+    default:
+        value = 0;
```

Comment:
I received RTE's latest specifications, and the expected default is 1

Reply:
ok